include/RAJA/policy/hip/atomic.hpp

```diff
@@ -166,401 +168,300 @@ RAJA_INLINE __device__ T hip_atomic_CAS_oper(T volatile *acc, OPER &&oper)
 }
 
 
+template < typename T, typename TypeList >
+struct is_any_of;
+
+template < typename T, typename... Types >
+struct is_any_of<T, list<Types...>>
+  : concepts::any_of<camp::is_same<T, Types>...>
+{};
+
+template < typename T, typename TypeList >
+using enable_if_is_any_of = std::enable_if_t<is_any_of<T, TypeList>::value, T>;
+
+template < typename T, typename TypeList >
+using enable_if_is_none_of = std::enable_if_t<concepts::negate<is_any_of<T, TypeList>>::value, T>;
+
```

Comment:
Does any of the code above belong in camp?

Reply:
That's probably a good helper to add with the concepts rework, I'll post up the minimal recut of that shortly and maybe we can pop it in.

Reply:
Did this make it into camp? or are we keeping it here?

Reply:
I didn't see it in main or the concepts refactor PR. Shall we bring this in and fix it later?

Reply:
deal!